src/utils/common.ts

```diff
@@ -128,12 +129,20 @@ export async function fetchJson<T = any>(
   return response.json();
 }
 
+/**
+ * Prepare human-readable name with protocol for an account.
+ * E.g.: `Æternity account 1`, `Bitcoin account 2`
+ */
+export function getDefaultAccountLabel({ protocol, idx }: { protocol?: Protocol, idx?: number }) {
+  return `${protocol || ''} ${tg('pages.account.heading')} ${(idx || 0) + 1}`;
```

Comment:
`protocol` = "aeternity" | "bitcoin". So it will not produce "Æternity". Are you sure this is correct?

Reply:
I'm sure it is not - will fix